lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -689,6 +747,20 @@ module Resources
                       'start': '2016-12-08T21:53:53.000Z'
```

Comment:
All of the relationship.id fields need to be updated since we no longer return two indexes.

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/644/commits/767fd43d02089bf605dfdde72918fc393bec5655#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR135 and places as well